src/Assets/TestProjects/AppWithSharedProject/SharedProject/SharedProject.projitems

```diff
@@ -1,7 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+    <MSBuildAllProjects Condition=" '$(MSBuildVersion)' == '' Or '$(MSBuildVersion)' &lt; '16.0' ">
```

Comment:
I am not sure if this has been discussed. But these files should not be used with MSBuild less than 16.0.

The CLI will require Dev16, which means if someone tries to use this in Dev15, they will have to do hacks to enable it and we don't support it.

And the CLI itself will carry MSBuild 16.

cc @rainersigwald